cmd/createcluster.go

```diff
@@ -119,6 +129,8 @@ func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.StringVar(&config.ClusterDir, "cluster-dir", ".charon/cluster", "The target folder to create the cluster in.")
 	flags.IntVarP(&config.NumNodes, "nodes", "n", 4, "The number of charon nodes in the cluster.")
 	flags.IntVarP(&config.Threshold, "threshold", "t", 3, "The threshold required for signature reconstruction. Minimum is n-(ceil(n/3)-1).")
+	flags.StringVar(&config.WithdrawalAddr, "withdrawal-address", defaultWithdrawalAddr, "Ethereum address to receive the returned stake and accrued rewards.")
+	flags.StringVar(&config.Network, "network", defaultNetwork, "Ethereum network to create validators for (default: mainnet). Options: mainnet, prater, kintsugi, kiln, gnosis.")
```

Comment:
Because of line 88, this is what I see for this command:

 ```
 --network string              Ethereum network to create validators for (default: mainnet). Options: mainnet, prater, kintsugi, kiln, gnosis. (default "prater")
 ```
 
 I guess remove the `(default: mainnet)` from the description string for now.